.github/workflows/coverity-scan.yml

```diff
@@ -0,0 +1,65 @@
+name: coverity-scan
+on:
+  schedule:
+    - cron: '0 9 * * 1,5' # Bi-weekly at 09:00 UTC on Monday and Thursday
```

Comment:
https://www.youtube.com/watch?v=txa5ia2w7-k

Reply:
love it